Initializing MyApi using builder giver Exception

Caused by: java.lang.IllegalStateException: No successful match so far
        at java.util.regex.Matcher.ensureMatch(Matcher.java:1116)
        at java.util.regex.Matcher.group(Matcher.java:382)
        at com.google.api.client.googleapis.GoogleUtils.<clinit>(GoogleUtils.java:66)
        at com.idragonit.backend.myApi.MyApi.<clinit>(MyApi.java:46) 
Version Using: implementation 'com.google.api-client:google-api-client:1.30.6'
in my andoid backed library
In GoogleUtils.java line number 64
Matcher versionMatcher = VERSION_PATTERN.matcher(VERSION);
I am getting VERSION string 0.0

--- a/google-api-client/pom.xml
+++ b/google-api-client/pom.xml
@@ -145,11 +145,11 @@
       <artifactId>commons-codec</artifactId>
       <scope>provided</scope>
       <version>1.14</version>
-   </dependency>
-   <dependency>
-     <groupId>com.google.guava</groupId>
-     <artifactId>guava</artifactId>
-   </dependency>
+    </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+    </dependency>
     <dependency>
       <groupId>androidx.annotation</groupId>
       <artifactId>annotation</artifactId>

--- a/google-api-client/pom.xml
+++ b/google-api-client/pom.xml
@@ -151,4 +151,5 @@
       <artifactId>guava</artifactId>
     </dependency>
   </dependencies>
+
 </project>

--- a/google-api-client/pom.xml
+++ b/google-api-client/pom.xml
@@ -146,10 +146,22 @@
       <scope>provided</scope>
       <version>1.14</version>
    </dependency>
+   <dependency>
+     <groupId>com.google.guava</groupId>
+     <artifactId>guava</artifactId>
+   </dependency>
     <dependency>
-      <groupId>com.google.guava</groupId>
-      <artifactId>guava</artifactId>
+      <groupId>androidx.annotation</groupId>
+      <artifactId>annotation</artifactId>
+      <version>1.1.0</version>
     </dependency>
   </dependencies>
-
+  
+  <repositories>  
+    <repository>
+      <id>google</id>
+      <name>Google Maven Repository</name>
+      <url>https://maven.google.com/</url>
+    </repository>
+  </repositories>
 </project>

--- a/google-api-client/src/main/java/com/google/api/client/googleapis/GoogleUtils.java
+++ b/google-api-client/src/main/java/com/google/api/client/googleapis/GoogleUtils.java
@@ -14,6 +14,7 @@
 
 package com.google.api.client.googleapis;
 
+import androidx.annotation.Keep;
 import com.google.api.client.util.SecurityUtils;
 import com.google.common.annotations.VisibleForTesting;
 import java.io.IOException;
@@ -30,6 +31,7 @@ import java.util.regex.Pattern;
  * @since 1.12
  * @author rmistry@google.com (Ravi Mistry)
  */
+@Keep
 public final class GoogleUtils {
 
   /** Current release version. */

